pages/talks.js

```diff
@@ -3,15 +3,15 @@ import React from 'react';
 import useDataApi from '../lib/useDataApi';
 import Layout from '../components/Layout';
 import Talk from '../components/Talk';
-import { CardWithLoading } from '../components/Card';
+import { CardWithLoading, CardWithError } from '../components/Card';
 import Grid from '../components/Grid';
 
-const LoadingCalendar = () =>
+const EmptyCalendar = ({ render }) =>
```

Comment:
`render` is the content?

Reply:
Yes, it's component. I will rename it `component`